myapp/urls.py

```diff
@@ -16,13 +16,87 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.contrib.auth import views as auth_views
+from .api import views
 
 from myapp.spa.views import SpaView
 from myapp.api.views import GreetingApi
 
+# turn off formatting by 'black'
+# fmt: off
 urlpatterns = [
-    path("admin/", admin.site.urls),
-    path("accounts/", include("django.contrib.auth.urls")),
-    path("api/greet", GreetingApi.as_view()),
-    path("", SpaView.as_view(), name="spa"),
```

Comment:
For some reason github doesn't seem to be updating these diffs could you remake this pr?